app/assets/stylesheets/application.css.scss

```diff
@@ -1,5 +1,6 @@
 @import "bootstrap-sprockets";
 @import "bootstrap";
+@import "bootstrap-extend";
```

Comment:
Good suggestion :smile: !  I'll choose to name it `shared/bootstrap-extension`, because it usually includes not only overrides but also extensions. I'll fix it later if no other objections.